DESCRIPTION

```diff
@@ -79,5 +79,9 @@ Collate:
     'argument_convention.R'
     'defaults.R'
     'jmpost-package.R'
+    'simulate_sf.R'
```

Comment:
Should this be called instead `simulations_sf.R`?

Reply:
Ah I see, if you merge this PR first and the SF PR later then it should be all fine moving this file to the SF branch